src/components/calendar/CalendarView.tsx

```diff
@@ -0,0 +1,199 @@
+import { makeStyles } from '@material-ui/core';
+import moment from 'moment';
+import React, { ComponentType, useContext, useEffect, useState } from 'react';
+import {
+  Calendar as BigCalendar,
+  CalendarProps,
+  momentLocalizer,
+  SlotInfo,
+  stringOrDate,
+  View,
+  Views,
+} from 'react-big-calendar';
+import withDragAndDrop from 'react-big-calendar/lib/addons/dragAndDrop';
+import { useHistory } from 'react-router';
+
+import { AppContext } from 'context/AppContext';
+import { ScheduledEventFilter } from 'generated/sdk';
+import { useQuery } from 'hooks/common/useQuery';
+
+import {
+  isSchedulerViewPeriod,
+  SchedulerViewPeriod,
+} from './CalendarViewContainer';
+import Event, { CalendarScheduledEvent, eventPropGetter } from './Event';
+import Toolbar from './Toolbar';
+
+moment.locale('en-gb');
+const localizer = momentLocalizer(moment);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function slotPropGetter(date: Date): any {
```

Comment:
I guess if you just remove return type `any` you can remove the eslint-disable-next-line, because return type would be inferred.

Reply:
Will to it 👍